util/src/main/java/org/cryptomator/util/crypto/BiometricAuthCryptor.java

```diff
@@ -50,7 +58,8 @@ public String encrypt(javax.crypto.Cipher cipher, String password) throws Illega
 	}
 
 	public String decrypt(javax.crypto.Cipher cipher, String password) throws IllegalBlockSizeException, BadPaddingException {
-		byte[] ciphered = cipher.doFinal(CipherImpl.getBytes(password.getBytes(StandardCharsets.ISO_8859_1)));
+		int ivLength = cryptoMode == CryptoMode.GCM ? CipherGCM.IV_LENGTH : CipherCBC.IV_LENGTH;
```

Comment:
Same as above: I'd use a switch-statement here.

Reply:
Fixed in https://github.com/cryptomator/android/commit/9615090966def68d10b86241c37ddace9946c0ef